src/DIRAC/Resources/Computing/PoolComputingElement.py

```diff
@@ -80,22 +76,16 @@ def _reset(self):
 
         self.processors = int(self.ceParameters.get("NumberOfProcessors", self.processors))
         self.ceParameters["MaxTotalJobs"] = self.processors
+        max_ram = int(self.ceParameters.get("MaxRAM", 0))
```

Comment:
Defining this at the CE level isn't going to work very well (e.g. the HLTFarm has several different hardware configurations with wildly different RAM per core).

Can we inspect the host to have a sensible default?

Reply:
This is already at the "inner CE" level (this is a very common confusion). So, basically it is at the level of the Worker Node.